larky/src/main/java/com/verygood/security/larky/modules/JsonModule.java

```diff
@@ -223,8 +224,8 @@ private void encode(Object x) throws EvalException {
         return;
       }
 
-      // e.g. tuple, list
-      if (x instanceof StarlarkIterable) {
+      // only real tuple, list JSON arrays
+      if (x instanceof StarlarkList || x instanceof Tuple) {
```

Comment:
Do we have a test for this? I understand why, but would this break behavior that folks are depending on?